src/transaction/log_applier.c

```diff
@@ -7043,141 +7043,175 @@ la_print_log_arv_header (const char *database_name, LOG_ARV_HEADER * hdr, bool v
+      if (replica_time_bound_str != NULL)
 	{
-	  replica_time_bound_str = prm_get_string_value (PRM_ID_HA_REPLICA_TIME_BOUND);
-	  db_datetime_to_string2 ((char *) timebuf, 1024, &ha_apply_info.log_record_time);
-
-	  printf ("\n *** Replica-specific Info. *** \n");
-	  if (replica_time_bound_str == NULL)
-	    {
-	      printf ("%-30s : %d second(s)\n", "Deliberate lag",
-		      prm_get_integer_value (PRM_ID_HA_REPLICA_DELAY_IN_SECS));
-	    }
-
-	  printf ("%-30s : %s\n", "Last applied log record time", timebuf);
-	  if (replica_time_bound_str != NULL)
-	    {
-	      printf ("%-30s : %s\n", "Will apply log records up to", replica_time_bound_str);
-	    }
+	  printf ("%-30s : %s\n", "Will apply log records up to", replica_time_bound_str);
 	}
     }
+
 check_applied_info_end:
-  if (error != NO_ERROR)
+
+  return error;
+}
+
+/*
+ * la_get_copied_log_info() - when applyinfo utility is executed with -L or -p option, this fuction will try to get copied log or copied log page information.
+ *   return: NO_ERROR or error code
+ *   database_name(in): db name
+ *   log_path(in): real path of log file
+ *   page_num(in): the number of log page 
+ *   verbose(in): applyinfo -v option
+ *   copied_eof_lsa(in/out): inforamtion for delayed log count
+ *   copied_append_lsa(in/out): current append location
+ */
+int
+la_get_copied_log_info (const char *database_name, const char *log_path, INT64 page_num, bool verbose,
+			LOG_LSA * copied_eof_lsa, LOG_LSA * copied_append_lsa)
+{
+  int error = NO_ERROR;
+  char active_log_path[PATH_MAX];
+  char log_path_buf[PATH_MAX];
+
+  assert (database_name != NULL);
+  assert (log_path != NULL);
+
+  if (realpath (log_path, log_path_buf) != NULL)
     {
-      printf ("%s\n", db_error_string (3));
+      log_path = log_path_buf;
+    }
+
+  memset (active_log_path, 0, PATH_MAX);
+  /* init la_Info */
+  la_init (log_path, 0);
+
+  printf ("\n *** Copied Active Info. *** \n");
+
+  fileio_make_log_active_name ((char *) active_log_path, la_Info.log_path, database_name);
+  if (!fileio_is_volume_exist ((const char *) active_log_path))
+    {
+      er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_LOG_MOUNT_FAIL, 1, active_log_path);
+      error = ER_LOG_MOUNT_FAIL;
+      goto check_copied_log_volume_info_end;
     }
-  error = NO_ERROR;
 
-  if (check_copied_info)
+  /* read copied active page */
+  error = la_find_log_pagesize (&la_Info.act_log, la_Info.log_path, database_name, false);
+  if (error != NO_ERROR)
     {
-      /* check active log file */
-      memset (active_log_path, 0, PATH_MAX);
-      fileio_make_log_active_name ((char *) active_log_path, la_Info.log_path, database_name);
-      if (!fileio_is_volume_exist ((const char *) active_log_path))
-	{
-	  er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_LOG_MOUNT_FAIL, 1, active_log_path);
-	  error = ER_LOG_MOUNT_FAIL;
-	  goto check_copied_info_end;
-	}
+      goto check_copied_log_volume_info_end;
```

Comment:
`check_copied_info_end` label name is changed to check_copied_log_volume_info_end.
And `check_copied_info_end` is changed to `check_copied_log_page_info_end`